src/Entity/CustomerBillingDataInterface.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\RefundPlugin\Entity;
+
+interface CustomerBillingDataInterface
+{
+    public function getId(): int;
```

Comment:
If we would like to be consistent with other getters, it should be `public function id(): int`